src/main/java/org/sopt36/ninedotserver/mandalart/service/command/SubGoalCommandService.java

```diff
@@ -89,6 +96,9 @@ private SubGoal getExistingSubGoal(Long subGoalId) {
             .orElseThrow(() -> new SubGoalException(SUB_GOAL_NOT_FOUND));
     }
 
+    private User getUserFrom(CoreGoal coreGoal) {
+        return coreGoal.getMandalart().getUser();
+    }
```

Comment:
CoreGoal 안에 getUser를 만들어서 메서드 체이닝을 줄이는 방안도 있을 것 같습니당!
```suggestion
// CoreGoal.java
private User getUserFrom() {
    return this.getMandalart().getUser();
}

// 이 부분
private User getCoreGoalUser(CoreGoal coreGoal) {
    return coreGoal.getCoreGoal();
}
```
근데 저도 이 부분이 좋다고 확실히 장담하지는 못하겠어서 한번 확인해보시고 적용하면 좋을 것 같아요!

Reply:
🫡🫡🫡